internal/cri/nri/nri_api_linux.go

```diff
@@ -93,6 +93,28 @@ func (a *API) RunPodSandbox(ctx context.Context, criPod *sstore.Sandbox) error {
 	return err
 }
 
+func (a *API) UpdatePodSandboxResources(ctx context.Context, criPod *sstore.Sandbox, overhead *cri.LinuxContainerResources, req *cri.LinuxContainerResources) error {
```

Comment:
A no-op version of this should be added to nri_api_other.go (should fix macOS/Windows build/linter failures).